content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md

```diff
@@ -81,6 +87,11 @@ possible and inform your users about this change and what steps they need to tak
   Kubernetes will freeze the legacy package repositories,
   (`apt.kubernetes.io` and `yum.kubernetes.io`).
   The freeze will happen immediately following the patch releases that are scheduled for September, 2023.
+- **12th January 2024:**
+  Kubernetes announced intentions to remove the legacy package repositories in January 2024
+- **4th March 2024:**
+  The legacy package repositories have been removed. It's not possible to install Kubernetes packages from
+  the legacy package repositories any longer
```

Comment:
```suggestion
- **12th January 2024:**  
  Kubernetes announced intentions to remove the legacy package repositories in January 2024
- **4th March 2024:**  
  The legacy package repositories have been removed. It's not possible to install Kubernetes packages from
  the legacy package repositories any longer
```

_the trailing whitespace is significant_

Reply:
Accepted the suggestion 👍